src/pages/Cart/Cart.js

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import CartInfoBox from './CartInfoBox/CartInfoBox';
+import CartDefault from './CartDefault/CartDefault';
+import './Cart.scss';
 
-function Cart() {
-  return <div>Cart1</div>;
-}
+export default function Cart() {
+  const [infoBox, setInfoBox] = useState([]);
 
-export default Cart;
+  useEffect(() => {
+    fetch('http://localhost:3000/data/cartData.json', {
```

Comment:
```suggestion
    fetch('/data/cartData.json', {
```

http://localhost:3000 은 생략 가능합니다. 포트 번호가 달라지면(ex. localhost:3001) 코드 전체를 바꿔야 하기 때문에 수정해주시기 바랍니다.

Reply:
적용완료했습니다!